zmon-common/src/main/java/org/zalando/zmon/domain/ServiceLevelStatus.java

```diff
@@ -36,6 +36,12 @@ public void setLastModifiedBy(String lastModifiedBy) {
 
 
     public static class ServiceLevelStatusData {
+        public static final String SECOND_LEVEL_WARNING = "Metrics visualization is currently only enabled for metrics classified as \"important\" or \"critical\" and is temporarily limited to a 12 hour span.";
```

Comment:
`12 hour span` is not fully correct, since it's actually only *last* 12 hours. I'd rephrase that to `... limited to the last 12 hours`.